src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
   <PropertyGroup Label="Configuration">
     <Id>Microsoft.Azure.Monitoring.SmartDetectors.Extensions</Id>
-    <Version>0.16.0.0</Version>
+    <Version>0.17.0.0</Version>
```

Comment:
>17 [](start = 15, length = 2)

There was no breaking or major change in the SmartDetectorsSDK. Consider just updating the minor version, to 0.16.1.0